src/librustc_codegen_ssa/back/linker.rs

```diff
@@ -712,6 +719,10 @@ impl<'a> Linker for MsvcLinker<'a> {
         }
     }
 
+    fn strip(&mut self) {
+        // noop
```

Comment:
For MSVC, you want `/DEBUG:NONE`. Note that the MSVC linker emits debug info as a separate PDB file, so it matters less than for GNU/Unix systems. However, some users still wish to avoid emission here: https://github.com/rust-lang/rust/issues/67012

Reply:
Hmmm, it looks like this might already be handled with `/DEBUG` argument above.

Reply:
Thanks for the suggestion. Do you think we can just add another argument to override the previous set option?